checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -169,6 +178,47 @@ private boolean isHandledArrayCreation(Tree argument, String receiver) {
         return false;
     }
 
+    /**
+     * Returns true iff {@code arg} is a reference to a {@code private final} array field whose
+     * initializer is `new T[0]`, `new T[] {}`, or `{}`.
+     */
+    private boolean isZeroLengthPrivateFinalFieldArray(ExpressionTree arg) {
+
```

Comment:
```suggestion
```

Nit: I don't like starting method bodies with an empty line... it seems like a waste to me.